Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1346,7 +1346,9 @@ def gen():
             futs = list(asyncio.as_completed(fs, loop=loop))
         self.assertEqual(len(futs), 2)
         waiter = asyncio.wait(futs)
-        done, pending = loop.run_until_complete(waiter)
+        # Deprecation from passing coros in futs to asyncio.wait()
+        with self.assertWarns(DeprecationWarning):
+            done, pending = loop.run_until_complete(waiter)
```

Comment:
The comment seemed necessary here because it would likely not be initially clear why this should cause a deprecation warning, particularly in the future when the tests have to be modified in 3.11. At a first glance, one may assume that `futs` is a list of futures based on it's name alone. However, `futs` is actually a list of completed coroutine objects:

```python
>>> import asyncio
>>> import asyncio.coroutines as coroutines
>>> a = asyncio.sleep(0)
>>> b = asyncio.sleep(0)
>>> fs = {a, b}
>>> futs = list(asyncio.as_completed(fs)
>>> for f in futs:
...   coroutines.iscoroutine(f)
... 
True
True
>>> for f in futs:
...    f
<coroutine object as_completed.<locals>._wait_for_one at 0x7f056fb4bdc0>
<coroutine object as_completed.<locals>._wait_for_one at 0x7f056fb80dc0>
```
This makes more sense when the exact behavior of `asyncio.as_completed()` is taken into consideration, but I think it would be reasonable to change the variable name `futs` into `completed_coros`. IMO, this would make it a lot less confusing at virtually no expense.

Admittedly, this took a fair bit of troubleshooting on my end before I realized what was going on. `asyncio.completed_coros()` is not exactly a function I use on a day-to-day basis, so I had not initially realized that `futs` was a list of completed coroutine objects.